lib/index.js

```diff
@@ -118,8 +122,29 @@ function addRoutes(app) {
     }
   }));
 
-  app.options('/verifier/presentations', cors());
-  app.post('/verifier/presentations', cors(), asyncHandler(async (req, res) => {
+  app.options(
+    ['/verifier/presentations', '/instances/:instanceId/presentations/verify'],
```

Comment:
Can we make the style here and the one on L144 consistent?

Reply:
Thanks, changed to make it consistent.